integration-tests/container/perf_event_open/sched_process_exit.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <asm/unistd.h>
+#include <linux/perf_event.h>
+#include <sys/ioctl.h>
+
+static long
+perf_event_open(struct perf_event_attr* hw_event, pid_t pid,
+                int cpu, int group_fd, unsigned long flags) {
+  int ret;
+
+  ret = syscall(__NR_perf_event_open, hw_event, pid, cpu,
+                group_fd, flags);
+  return ret;
+}
+
+int main(int argc, char** argv) {
+  struct perf_event_attr pe;
+  long long count;
+  int fd;
+
+  // How long to wait for events, in seconds
+  int wait_interval = 10;
+
+  // /sys/kernel/debug/tracing/events/sched/sched_process_exit/id
+  int tracepoint_code = 310;
+
+  if (argc == 3) {
+    wait_interval = atoi(argv[1]);
+    tracepoint_code = atoi(argv[2]);
+  }
```

Comment:
Looks like we are always using the arguments passed in, so we might want to fail if the number of arguments is wrong instead.
```suggestion
  if (argc != 3) {
    fprintf(stderr, "Expected 2 arguments, got: %d", argc - 1);
    exit(EXIT_FAILURE);
  }
  // How long to wait for events, in seconds
  int wait_interval = atoi(argv[1]);

  // /sys/kernel/debug/tracing/events/sched/sched_process_exit/id
  int tracepoint_code = atoi(argv[2]);

```